controllers/api/request-routes.js

```diff
@@ -93,42 +95,43 @@ router.get("/:id", (req, res) => {
       id: req.params.id,
     },
     attributes: ["id", "request_message"],
-    include: [
-      {
-        model: Property,
-        attributes: [
-          "id",
-          "bedrooms",
-          "bathrooms",
-          "rent",
-          "sq_ft",
-          "address_line1",
-          "address_line2",
-          "city",
-          "state",
-          "zip_code",
-          "county",
-        ],
-        include: [
-          {
-            model: PropertyType,
-            attributes: ["id", "category", "description"],
-          },
-          {
-            model: RequestType,
-            attributes: ["id", "category", "description"],
-          },
-          {
-            model: StatusType,
-            attributes: ["id", "category", "description"],
-          },
-          {
-            model: WorkOrderType,
-            attributes: ["id", "category", "description"],
-          },
-        ],
-      },
-    ],
+    include: [{ all: true, nested: true }],
```

Comment:
I would suggest changing the nested to false.  It doesnt return everything like it does with it being true
```javascript
   include: [{ all: true, nested: false }]
```